contracts/CompoundGovernor.sol

```diff
@@ -1,5 +1,199 @@
+        IVotes _compAddress,
+        TimelockControllerUpgradeable _timelockAddress,
+        uint48 _initialVoteExtension,
+        address _initialOwner
+    ) public initializer {
+        __Governor_init(_name);
+        __GovernorSettings_init(_initialVotingDelay, _initialVotingPeriod, _initialProposalThreshold);
+        __GovernorVotes_init(_compAddress);
+        __GovernorTimelockControl_init(_timelockAddress);
+        __GovernorPreventLateQuorum_init(_initialVoteExtension);
+        __Ownable_init(_initialOwner);
+    }
+
+    /// @inheritdoc GovernorTimelockControlUpgradeable
+    /// @dev We override this function to resolve ambiguity between inherited contracts.
+    function _cancel(
+        address[] memory _targets,
+        uint256[] memory _values,
+        bytes[] memory _calldatas,
+        bytes32 _descriptionHash
+    ) internal virtual override(GovernorTimelockControlUpgradeable, GovernorUpgradeable) returns (uint256) {
+        return GovernorTimelockControlUpgradeable._cancel(_targets, _values, _calldatas, _descriptionHash);
+    }
+
+    /// @inheritdoc GovernorPreventLateQuorumUpgradeable
+    /// @dev We override this function to resolve ambiguity between inherited contracts.
+    function _castVote(
+        uint256 _proposalId,
+        address _account,
+        uint8 _support,
+        string memory _reason,
+        bytes memory _params
+    ) internal virtual override(GovernorPreventLateQuorumUpgradeable, GovernorUpgradeable) returns (uint256) {
+        return GovernorPreventLateQuorumUpgradeable._castVote(_proposalId, _account, _support, _reason, _params);
+    }
+
+    /// @dev Executes a proposal after it has been queued.
+    /// @dev We override this function to resolve ambiguity between inherited contracts.
+    /// @param _proposalId The id of the proposal.
+    /// @param _targets A list of target addresses for calls to be made in the proposal.
+    /// @param _values A list of values (ETH) to be passed to the calls in the proposal.
+    /// @param _calldatas A list of calldata for the calls in the proposal.
+    /// @param _descriptionHash The hash of the description for the proposal.
+    function _executeOperations(
+        uint256 _proposalId,
+        address[] memory _targets,
+        uint256[] memory _values,
+        bytes[] memory _calldatas,
+        bytes32 _descriptionHash
+    ) internal virtual override(GovernorTimelockControlUpgradeable, GovernorUpgradeable) {
+        return GovernorTimelockControlUpgradeable._executeOperations(
+            _proposalId, _targets, _values, _calldatas, _descriptionHash
+        );
+    }
+
+    /// @dev returns executor address
+    /// @return address of the executor.
+    function _executor()
+        internal
+        view
+        override(GovernorTimelockControlUpgradeable, GovernorUpgradeable)
+        returns (address)
+    {
+        return address(this);
+    }
+
+    /// @dev Queues a proposal to be executed after it has succeeded.
```

Comment:
can we `@inheritdoc GovernorTimelockControlUpgradeable` this?

Reply:
Done: dbee29e9efddfcb7851b909f8e9b1e12d7640fe0